server/routes/serviceProviderReferrals/scheduleAppointmentPresenter.ts

```diff
@@ -68,11 +69,11 @@ export default class ScheduleAppointmentPresenter {
     return config.features.npsOfficeLocationSelection.enabled
   }
 
-  get isInitialAssessmentAppointment(): boolean {
-    return this.currentAppointment === null
+  get allowSessionTypeSelection(): boolean {
+    return this.formType !== 'supplierAssessment'
   }
 
-  readonly initialAssessmentSessionType = 'ONE_TO_ONE'
+  readonly sessionTypeWhenSelectionNotAllowed = 'ONE_TO_ONE'
```

Comment:
I think the idea I was trying to convey behind the previous variable name was that all initial assessment appointments are 'ONE_TO_ONE', which is the reason we don't show a session type - I feel like it was possibly more explicit than `whenSelectionNotAllowed`, what do you think?

Reply:
So, I was thinking about the interface that the presenter exposes to the view. I chose the naming so that the view doesn't need to know about business logic (i.e. that an initial assessment appointment is distinguished in some way), and all it needs to know is whether to show the session type controls and which value to use in that case. What do you think?

Reply:
Yeah, I suppose that's a fair point - the view doesn't need to know that logic, I wonder if there's some other place we can document that logic though, is there anywhere we do that at the moment?

Reply:
I don't think we do – I've added a comment above the variable here.